src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -183,7 +183,7 @@ export class QuickBar extends LitElement {
         .item=${item}
         index=${ifDefined(index)}
         hasMeta
-        graphic=${item.altText ? "avatar" : "icon"}
+        graphic="avatar"
```

Comment:
Yeah, just tested locally and you're right. This is more visually consistent. Good catch